package.json

```diff
@@ -9,23 +9,24 @@
     "build": "npm run build:es && npm run build:cjs-es5",
     "build:es": "tsc -p .",
     "build:cjs-es5": "tsc -p . --outDir lib/cjs-es5 --module commonjs --target es5",
-    "build:demo": "npm run dist && cp dist/kinto-http.min.js demo/kinto-http.min.js && cp dist/kinto-http.min.js.map demo/kinto-http.min.js.map",
+    "build:demo": "npm run dist && shx cp dist/kinto-http.min.js demo/kinto-http.min.js && shx cp dist/kinto-http.min.js.map demo/kinto-http.min.js.map",
     "build:readme": "./node_modules/.bin/toctoc -w -d 2 README.md",
-    "cs-check": "prettier -l '{src,test}/**/*.js'",
-    "cs-format": "prettier '{src,test}/**/*.js' --write",
+    "cs-check": "prettier -l \"{src,test,bin}/**/*.{js,ts}\"",
```

Comment:
What did motivate this change? Do you find `\"` more readable than `'` or is it because Windows shell doesn't handle `'`? :fearful:

Reply:
The Windows command prompt doesn't treat single quote characters the same way as double quotes unfortunately.